TestNodeGetCapabilities unit test fails as expectation changed

/kind bug

**What happened?**
TestNodeGetCapabilities unit test fails as expectation changed
```
=== RUN   TestNodeGetCapabilities
    /Users/madhankumar/work/csie2etest/prowFix/f2/ibm-powervs-block-csi-driver/pkg/driver/node_test.go:812: Expected response {capabilities:<rpc:<type:STAGE_UNSTAGE_VOLUME > > capabilities:<rpc:<type:EXPAND_VOLUME > > }, got {capabilities:<rpc:<type:STAGE_UNSTAGE_VOLUME > > capabilities:<rpc:<type:EXPAND_VOLUME > > capabilities:<rpc:<type:GET_VOLUME_STATS > > }
--- FAIL: TestNodeGetCapabilities (0.00s)
```

Unit test was not updated when GetVolumeStatus feature was added.

**What you expected to happen?**
Unit test should pass

**How to reproduce it (as minimally and precisely as possible)?**
`go test -v pkg/driver/node_test.go`

--- a/pkg/driver/node_test.go
+++ b/pkg/driver/node_test.go
@@ -796,6 +796,13 @@ func TestNodeGetCapabilities(t *testing.T) {
 				},
 			},
 		},
+		{
+			Type: &csi.NodeServiceCapability_Rpc{
+				Rpc: &csi.NodeServiceCapability_RPC{
+					Type: csi.NodeServiceCapability_RPC_GET_VOLUME_STATS,
+				},
+			},
+		},
 	}
 	expResp := &csi.NodeGetCapabilitiesResponse{Capabilities: caps}
 

